beacon_node/beacon_chain/src/events.rs

```diff
@@ -1,6 +1,9 @@
+use bus::Bus;
 use serde_derive::{Deserialize, Serialize};
+use slog::{error, Logger};
 use std::marker::PhantomData;
-use types::{Attestation, Epoch, EthSpec, Hash256, SignedBeaconBlock};
+use std::sync::{Arc, Mutex};
```

Comment:
AFAIK, all the other `Mutex` in Lighthouse use `rwlock::Mutex`. They're supposed to be faster and they also have the nice property that they don't return `Error`s on `lock()`. Not a big deal, but might be nice to stay consistent.